.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please delete the .vscode folder and add it to .gitignore as it is specific to the user's machine